src/test/java/LoginTests.java

```diff
@@ -110,4 +108,16 @@ public static void loginValidEmailEmptyPasswordTest() {
 
         driver.quit();
     }
+
+    @Test
+    public void loginValidCredentials(){
+        LoginPage loginPage = new LoginPage(driver);
+
+        loginPage.enterEmail();
+        loginPage.enterPassword();
+        loginPage.clickSubmit();
```

Comment:
Add, Asser

One Test - One Assert

For example

![Clean_Test](https://github.com/dossantos-e/A45-QA-Automation-Framework/assets/104761482/d5f19aab-0dde-4c3e-87f8-dcb9f05d95f4)